test/test_integrator.m

```diff
@@ -1,4 +1,4 @@
-function [results,stats,model,problem_options] = test_integrator(use_fesd, rk_representation, rk_scheme, dcs_mode)
```

Comment:
is there are reason why the names of the tests here in model follow a different format than for other classes?

Reply:
In what sense? If you mean the structure of the `model` and `dcs` tests: I treat those as more "unit" tests than the ones in the root of the `test` directory. As such I think it is ok to have a little bit more verbose structure with the name of the test suggesting what exactly is being tested.

Reply:
Note these functions are just helpers. The test framework runs all of the classes which are subclasses of `TestCase` in a given folder of which e.g. `TestIntegrator` is one. These then call their corresponding helper functions. The functions themselves are ignored by the matlab test framework.